scim-sdk-server/src/main/java/de/captaingoldfish/scim/sdk/server/patch/PatchResourceHandler.java

```diff
@@ -89,24 +89,57 @@ public boolean addResourceValues(ObjectNode resource, JsonNode readJsonDocument,
       else
       {
         SchemaAttribute schemaAttribute = getSchemaAttribute((extensionUri == null ? "" : extensionUri + ":") + key);
-        verifyImmutableAndReadOnly(resource, schemaAttribute);
+        boolean resourceChanged;
+        boolean isReadOnly = isReadOnly(resource, schemaAttribute);
```

Comment:
please rename fielt to:

isReadOnlyAndSet

Reply:
👍